### PR TITLE
Hotfix 

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by John-athan14 — DISMISSED** [review]

File: jsconfig.json [hunk 1]
```diff
@@ -2,5 +2,10 @@
   "compilerOptions": {
     "baseUrl": "src"
   },
-  "include": ["src"]
+  "include": [
+    "src",
+    "SectionSearchbar/SectionSearch.jsx",
+    "SectionSearchbar/SearchBar",
+    "SectionSearchbar/RotateLogo"
```
Comment (LoicBrassart) [inline jsconfig.json]: What does this do ?

File: src/App.css [hunk 2]
```diff
@@ -31,12 +31,12 @@
...
+/* h1,
 h2,
 h3,
 h4,
 h5,
 h6 {
   font-family: var(--strong-font-family);
   text-transform: uppercase;
-}
+} */
```
Comment (LoicBrassart) [inline src/App.css]: Dead code

File: src/Components/Header/Header.css [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
+  margin: 3px 10px 0 5px;
+  cursor: pointer;
 }
 
-.filter {
+.nav-logo {
+  width: 150px;
+}
+
+/* .filter {
```
Comment (LoicBrassart) [inline src/Components/Header/Header.css]: Dead code
.
.
.

File: src/Components/Header/Header.css [hunk 4]
```diff
@@ -30,9 +32,9 @@
 .deck {
   width: 3vw;
   margin-right: 1.5vw;
-}
+} */
```
Comment (LoicBrassart) [inline src/Components/Header/Header.css]: .
.
.
Down to here

File: src/Components/Header/Header.css [hunk 5]
```diff
@@ -30,9 +32,9 @@
...
 
-@media only screen and (max-width: 600px) {
+/* @media only screen and (max-width: 600px) {
```
Comment (LoicBrassart) [inline src/Components/Header/Header.css]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Beware of your commit messages, they're not-so-useful without reading the full code ;-)

File: jsconfig.json (hunk 1 above)
Comment (NacymeG) [reply jsconfig.json]: I don't know, it was added automatically...

**Review by John-athan14 — DISMISSED** [review]
ok

File: jsconfig.json [hunk 6]
```diff
@@ -2,5 +2,10 @@
   "compilerOptions": {
     "baseUrl": "src"
   },
-  "include": ["src"]
+  "include": [
```
Comment (Hacene010) [inline jsconfig.json]: Remove it

File: src/Components/SectionSearchbar/SearchBar/SearchBar.jsx [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+export default function SearchBar() {
+  return (
+    <>
+      <h1>Planeswalker</h1> <br />
+      <h2>Magic: The Gathering Database</h2>
+      <div className="wrap">
+        <input type="search" className="searchTerm" placeholder="Search..." />
+        <img
+          className="input-logo"
+          alt="loupe"
```
Comment (Hacene010) [inline src/Components/SectionSearchbar/SearchBar/SearchBar.jsx]: No french words plz

File: src/Components/SectionSearchbar/SectionSearch.css [hunk 8]
```diff
@@ -1,8 +1,7 @@
 .Section_searchbar {
```
Comment (Hacene010) [inline src/Components/SectionSearchbar/SectionSearch.css]: Use camelCase for your classNames everywhere or snack_case everywhere to be coherent

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some changes -> see the comments

**Review by John-athan14 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil!

**Review by Hacene010 — APPROVED** [review]
that's better ; )

**Pull request merged**